### DISCUSSION TITLE
Adding a profile avatar

### DISCUSSION THREAD

**Discussion by InfiniteVerma** [docker-mailserver/docker-mailserver#2821] [Q&A]
Is it possible to add a profile picture to each user so that that's visible on emails sent/received on clients? (Similar to how gmail does it?)

Thank you!

**Comment by casperklein** [comment]
[That is not possible - it's an gmail feature](https://stackoverflow.com/questions/26938529/avatar-for-an-email-address).

**Comment by BaileyJM02** [comment]
LDAP is what you would want to look into. Not 100% sure if it has a solution but I think most providers support checking for the `jpegPhoto` parameter. 

This [Gitea Issue](https://github.com/go-gitea/gitea/issues/4144) has some useful discussion regarding the matter and may be useful to at least offer context into what to research.

**Reply by BaileyJM02** [reply]
A useful schema resource: https://ldapwiki.com/wiki/JpegPhoto
